src/NLog.StructuredLogging.Json.Tests/NLog.StructuredLogging.Json.Tests.csproj

```diff
@@ -19,21 +18,24 @@
     <Content Include="nlog.config">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-  </ItemGroup> 
+  </ItemGroup>
   <ItemGroup>
     <PackageReference Include="FakeItEasy" Version="3.4.2" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
     <PackageReference Include="NUnit" Version="3.7.1" />
     <PackageReference Include="NUnitLite" Version="3.7.1" />
-  </ItemGroup> 
+    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0" />
+    <PackageReference Include="Mono.Cecil" Version="0.10.0-beta6" />
```

Comment:
@AnthonySteele Good question, I probably need a comment in there, it fails for me when I don't have this, it's a known issue with the new NUnit test adapter, this is a workaround. I'll dig out a link